File: documentation/user/en/query/requirements/examples/histogram/price-histogram.cs
EvitaResponse<ISealedEntity> entities = evita.QueryCatalog(
	"evita",
	session => session.QuerySealedEntity(
        Query(
        	Collection("Product"),
        	FilterBy(
        		PriceInPriceLists("basic"),
        		PriceInCurrency(new Currency("EUR")),
        		PriceValidInNow()
        	),
        	Require(
        		PriceHistogram(20, Standard)
        	)
        )
	)
);

File: documentation/user/en/query/requirements/examples/histogram/price-histogram.java
final EvitaResponse<SealedEntity> entities = evita.queryCatalog(
	"evita",
	session -> {
		return session.querySealedEntity(
			query(
				collection("Product"),
				filterBy(
					priceInPriceLists("basic"),
					priceInCurrency(Currency.getInstance("EUR")),
					priceValidInNow()
				),
				require(
					priceHistogram(20, STANDARD)
				)
			)
		);
	}
);